cmd/urfave-cli-genflags/main.go

```diff
@@ -163,27 +169,33 @@ func runGenFlags(cCtx *cli.Context) error {
 		return err
 	}
 
-	genTestBuf := &bytes.Buffer{}
-	if err := genTestTmpl.Execute(genTestBuf, spec); err != nil {
-		return err
-	}
-
 	if err := os.WriteFile(cCtx.Path("generated-output"), genBuf.Bytes(), 0644); err != nil {
 		return err
 	}
 
-	if err := os.WriteFile(cCtx.Path("generated-test-output"), genTestBuf.Bytes(), 0644); err != nil {
-		return err
-	}
-
 	if _, err := sh(cCtx.Context, cCtx.Path("goimports"), "-w", cCtx.Path("generated-output")); err != nil {
 		return err
 	}
 
-	if _, err := sh(cCtx.Context, cCtx.Path("goimports"), "-w", cCtx.Path("generated-test-output")); err != nil {
-		return err
-	}
+	if !cCtx.IsSet("altsrc") {
```

Comment:
nit: invert this `if` condition and `return nil` instead of nesting the production code inside the `if` condition; wdyt?

Reply:
Yep makes sense. Thanks